.changeset/rich-beers-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-openlmis': minor
+---
+
+Add generic helpers. `request()`, `post()`, `get()` and `put()`
```

Comment:
We don't need a changeset here because this is the first release of this adaptor

Just copy this line into `CHANGELOG.md`

Reply:
Done ✅ , Thanks 👍🏽